### PR TITLE
Add trigger for julia gc

### PR DESCRIPTION
**Pull request by pivnicek**

### CODE REVIEW

File: main.php [hunk 1]
```diff
@@ -34,3 +34,9 @@
 $process->setTimeout(null);
 $process->mustRun();
 print $process->getOutput() . "\n";
+
+$command = 'curl -X "POST" "' . $syrupUrl . '/provisioning/manage/server/docker/garbage-collection?type=julipyter&hours=120" -H "X-KBC-ManageApiToken: ' . $token . '"';
```
Comment (odinuv) [inline main.php]: the hours parameter doesn't exist any more

**Review by odinuv — APPROVED** [review]
must be deployed after this https://github.com/keboola/provisioning-router/pull/180 is deployed

**Comment by pivnicek** [comment]
tak jsem smazal `hours` vsude a https://travis-ci.com/keboola/provisioning-router/builds/131947064 tak @odinuv jdu mergnout  cajk?

**Comment by odinuv** [comment]
ja, schon

**Pull request merged**